packages/core/src/types/IImageCalibration.ts

```diff
@@ -0,0 +1,38 @@
+import CalibrationTypes from '../enums/CalibrationTypes';
+
+/**
+ * IImageCalibration is an object that stores information about the type
+ * of image calibration.
+ */
+export interface IImageCalibration {
+  /** The pixel spacing for the image, in mm between pixel centers */
+  rowPixelSpacing?: number;
+  columnPixelSpacing?: number;
+
+  /** The scaling of this image - new spacing = original pixelSpacing/scale */
+  scale?: number;
+
+  /** The type of the pixel spacing, distinguishing between various
+   * types projection (CR/DX/MG) spacing and volumetric spacing (the type is
+   * an empty string as it doesn't get a suffix, but this distinguishes it
+   * from other types)
+   */
+  type: CalibrationTypes;
+
+  // A tooltip which can be used to explain the calibration information
+  tooltip?: string;
+  /**
+   * Boolean to indicate that the image has spacing information (the pixelSpacing
+   * can just be 1,1 to use image pixels, but this
+   * isn't distinguishable from a real spacing of 1mm square pixels so a real
+   * flag here is needed.)
+   */
+  hasPixelSpacing?: boolean;
+  /** Indication of projection (eg X-Ray type) spacing and volumetric type */
+  isProjection?: boolean;
+  // The DICOM defined ultrasound regions.  Used for non-distance spacing
```

Comment:
please be consistent in using /** or // , also please remove the empty lines

Reply:
I'm going to try to use /** consistently as that is processed by the tsdoc parser.  In other files, I will update the newly added items but not things in other classes/types.